tests/tests_use_cases/test_use_case_topology.py

```diff
@@ -0,0 +1,236 @@
+        time.sleep(15)
+        
+        # Step 4: Verify topology is updated with link status down
+        response = requests.get(api_url)
+        assert response.status_code == 200, response.text
+        updated_topology = response.json()
+        
+        # Find the specific link in the topology
+        intra_domain_link_id = "urn:sdx:link:tenet.ac.za:Tenet01/2_Tenet03/2"
+        links = {link["id"]: link for link in updated_topology["links"]}
+        assert intra_domain_link_id in links, f"Link {intra_domain_link_id} not found in topology"
+        assert links[intra_domain_link_id]["status"] == "down", f"Link status is not down: {links[intra_domain_link_id]}"
+        
+        # Step 5: Verify L2VPN status is updated to down
+        response = requests.get(l2vpn_api_url)
+        assert response.status_code == 200, response.text
+        l2vpn_status = response.json().get(l2vpn_id).get("status")
+        assert l2vpn_status == "down", f"L2VPN status should be down, but is {l2vpn_status}"
+        
+        # test connectivity
+        assert ', 100% packet loss,' in h6.cmd('ping -c4 10.1.1.8')
+
+        # Step 6: Verify no topology changes were made (number of nodes and links should be the same)
+        assert len(initial_topology["nodes"]) == len(updated_topology["nodes"]), "Number of nodes changed"
+        assert len(initial_topology["links"]) == len(updated_topology["links"]), "Number of links changed"
+
+    @pytest.mark.xfail(reason="Connectivity is not verified with a PING test between hosts after creating an L2VPN")
+    def test_011_intra_domain_link_down_path_found(self):
+        """
+        Test Use Case 1: Intra-domain Link Status Change (Down)
+        
+        Verify that when an intra-domain link goes down:
+        - SDX Controller updates the SDX topology with the link status
+        - SDX Controller updates affected L2VPN statuses
+        - No topology or service changes are made
+
+        When possible, the SDX-Controller can find a path based on the exported topology.
+        When not possible, the SDX Controller will only update the L2VPN’s status using the means available.
+        """
+        
+        # Step 1: Get initial topology and create a test L2VPN that uses the intra-domain link
+        api_url = SDX_CONTROLLER + '/topology'
+        response = requests.get(api_url)
+        assert response.status_code == 200, response.text
+        initial_topology = response.json()
+        
+        # Create a L2VPN that will use the intra-domain link
+        l2vpn_api_url = SDX_CONTROLLER + '/l2vpn/1.0'
+        l2vpn_payload = {
+            "name": "Test L2VPN for intra-domain link",
+            "endpoints": [
+                {
+                    "port_id": "urn:sdx:port:tenet.ac.za:Tenet01:50",
+                    "vlan": "400",
+                },
+                {
+                    "port_id": "urn:sdx:port:tenet.ac.za:Tenet02:50",
+                    "vlan": "400",
```

Comment:
by experience, to minimize the changes of one failing test interfere with the other, you should use different VLANs here.